build.gradle

```diff
@@ -20,3 +20,47 @@ project.ext {
         RELEASE_KEY_PASSWORD = "*****"
     }
 }
+
+// Copies unit test reports to the 'artifacts' directory
+task copyUnitTestBuildArtifacts << { task ->
+    def hashPipe = new ByteArrayOutputStream()
+    task.project.exec {
+        commandLine = ['git', 'rev-parse', '--verify', 'HEAD']
+        standardOutput = hashPipe
+    }
+
+    def date = new Date()
+    def formattedDate = date.format('yyyyMMdd-HHmmss')
```

Comment:
I don't think we need the date? If we need it for reference, it should be on the associated travis build page. The way I have things set up, we'll already go under a folder specific to our hash and build number on S3. It might be good to write the date to a file just in case though